src/main/java/step_2/Numbers.java

```diff
@@ -0,0 +1,51 @@
+package step_2;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Numbers {
+
+    public static class Number {
+
+        private final int value;
+
+        public Number(String text) {
+            this(Integer.parseInt(text));
+        }
+
+        private Number(int value) {
+            toIntAndCheckNegativeNumber(value);
+            this.value = value;
+        }
+
+        private void toIntAndCheckNegativeNumber(int value) {
+            if (value < 0) {
+                throw new IllegalArgumentException("음수는 허용되지 않습니다.");
+            }
+        }
+
+        public int getValue() {
+            return this.value;
+        }
+    }
+
+    private final Number[] numbers;
+
+    public Numbers(Collection<String> chars) {
+        this.numbers = chars.stream().map(Number::new).toArray(Number[]::new);
+    }
+
+    public List<Integer> getValues() {
+        return Arrays.stream(this.numbers).map(Number::getValue).collect(Collectors.toList());
```

Comment:
```suggestion
        return Arrays.stream(this.numbers)
        .map(Number::getValue)
        .collect(Collectors.toList());
```

스트림은 내려쓰는 것이 관례상 좋습니다.

Reply:
@seok-2-o 코드 라인 수를 최소화하려고 했던 작업인데, 다음에는 관례를 따르도록 노력하겠습니다!